content/docs/user-guide/assets/scene-settings/actors-tab.md

```diff
@@ -0,0 +1,104 @@
+| **Use Advanced Settings** | When active, this property exposes the **Advanced coordinate system change** properties. |
+
+### Advanced coordinate system change
+
+![The Scene Settings Actors tab Coordinate system change modifier advanced settings.](/images/user-guide/assets/scene-settings/coordinate-system-change-modifier-2.png)
+
+Change the **Translation** (position), **Rotation** (orientation), and **Scale** of a mesh relative to how it was authored.
+
+| Property | Description |
+| - | - |
+| **Relative Origin Node** | Select the source asset node that serves as the origin for the actor group. The actor group is transformed relative to the selected node. The node does not need to be part of the actor group. Generally, the `RootNode` of the source asset is selected here as it represents the scene origin of the source asset. There are cases, however, where it might be preferable to apply the transform relative to a bone or mesh in the source file that is not the `RootNode`. |
+| **Translation** | Set the position offset of the product asset. This property is most commonly used to center the product asset or align the product asset with the ground.  |
+| **Rotation** | Set the orientation offset of the product asset in degrees. This property is most commonly used to reorient a product asset to O3DE's coordinate system. |
+| **Scale** | Set the scale offset of the product asset. O3DE's base unit is one meter. This property is most commonly used to uniformly scale product assets from DCC applications that use centimeters or inches as base units. |
+
+## Scale actor
+
+![The Scene Settings Actors tab Scale actor modifier.](/images/user-guide/assets/scene-settings/scale-actor-modifier.png)
+
+Set a uniform scale for the actor group. This setting is useful if the source asset is created in a DCC application that uses a different base standard unit of measurement than O3DE.
+
+| Property | Description |
+| - | - |
+| **Scale factor** | The actor is uniformly scaled by this value. |
+
+## Skeleton LOD
+
+![The Scene Settings Actors tab Skeleton lod modifier.](/images/user-guide/assets/scene-settings/skeleton-lod-modifier.png)
+
+A Skeleton LOD is a simplified skeleton where some leaf bones have been removed from the skeleton. The farther an entity is from the camera, the less detail is required from the actor. Using a character as an example, fine details such as facial animation and individual finger bones may not be required beyond a certain distance from the camera. The corresponding bones for these details can be optimized out of the skeleton. In networked games, the skeletons used on dedicated servers can be optimized down to the bones that transform PhysX colliders that provide hit detection and triggers.

```

Comment:
This is a bit odd because there might be bones in the chain that aren't removed because they aren't leaf bones, but also aren't directly linked to colliders. Will come up with something more clear and check it in.